### ISSUE TITLE
Fan speed does not reduce after temps fall below threshold

### ISSUE THREAD

**Issue by cmbail** [cjbdev/TPFanControl#11]
As per your default .ini spec - fans come on only at 60 C. Fans run just enough to keep temps below 70 C when I run a stress test. However, after temps drop below 60 C, fans don't stop, even now at 38 C. Confirmed I'm in your default Smart Mode. Odd because app windows reports it did set fan control to 0x00 (screenshot). I definitely hear the fan (could be just one). Is it possible the app is only controlling one of the two fans?

![fancontrol_2019-12-02_13-24-51](https://user-images.githubusercontent.com/3613239/69984550-59934700-1507-11ea-8bac-0216e496aea8.png)

**Comment by byrnes** [comment]
What device is this on? Also, make sure you don't have other versions of TPFC running in the background.

**Comment by cmbail** [comment]
ThinkPad X1 Extreme Gen 2. 

Definitely only one instance of "fancontrol.exe" running on the machine. (And only one exists on the drive.)

**Comment by AizenSousuke** [comment]
@cmbail did you get a solution for this?

**Comment by cmbail** [comment]
@AizenSousuke No, and switched to different hardware, so I stopped exploring it.
